### ISSUE TITLE
[docs] - need to update app icon , splash icon generator Figma File

### ISSUE THREAD

**Issue by adsalihac** [expo/expo#32809]
### Summary

Due to latest expo SDK 52 , expo providing iOS tint icon , splash icon .etc , current Figma file is outdated.

**Better for Everyone**

- App Icon - android , iOS.
- Splash
- Fav Icon
- Notification

Above case need to mange Dark , Light and Tint .etc is better.

### Link to the related docs page

https://www.figma.com/community/file/1155362909441341285

**Comment by brentvatne** [comment]
@adsalihac - we plan on it! this was a lot to have in time for the release, but in the coming weeks we should be able to get around to this

**Comment by adsalihac** [comment]
> @adsalihac - we plan on it! this was a lot to have in time for the release, but in the coming weeks we should be able to get around to this

Thank you

**Comment by adsalihac** [comment]
I have a question regarding the possibility of managing all cases using a single dark and light icon.

I am uncertain about the feasibility of this approach. However, I am curious about the current usage of icons for splash screens, iOS tint and dark case, and Android adaptive icons. These icons primarily differ in size and padding adjustments.

**use these 3 Icon Type for app.json**
- icon - with bg. if existing this , take this.
- dark icon png with transparent
- light icon png with transparent.

**Icon Types**
- App Icon - android , iOS.
- Splash
- Fav Icon
- Notification

I am unfamiliar with the process of generating icons in Expo. Additionally, I am uncertain whether this proposal is sound or not. thank you. @brentvatne

**Comment by enkyuan** [comment]
seconding this! i noticed that the aspect ratio on the icon is a bit weird right now (maybe due to resizing issues) but would love to see the updated figma files so i can update the splash and icon images.

**Comment by fobos531** [comment]
@brentvatne Seconding this! Also it seems there seems to be conflicts in icon generation on Android between splash assets and adaptive icons. If you have an adaptive icon config specified as well as a splash config via the new config plugin, those two seem to completely collide and the resulting launcher icon is the overlap of the two.

**Comment by alanjhughes** [comment]
@fobos531 I've just tried to repro this using this config
```json
{
  "expo": {
    "android": {
      "adaptiveIcon": {
        "foregroundImage": "./assets/adaptive-icon.png",
        "backgroundColor": "#232323"
      },
    },
    "plugins": [
      [
        "expo-splash-screen",
        {
          "image": "./assets/splash-icon.png",
          "resizeMode": "contain",
          "backgroundColor": "#232323",
          "imageWidth": 200
        }
      ]
    ]
  }
}
```
but I'm not seeing an issue. Can you provide an example config I can use to repro this?

**Comment by alanjhughes** [comment]
@adsalihac I've tried that now but I'm not sure what issue I should see? Splashscreen is created correctly

**Comment by adsalihac** [comment]
> @adsalihac I've tried that now but I'm not sure what issue I should see? Splashscreen is created correctly

Actually i am using [bootsplash](https://www.npmjs.com/package/react-native-bootsplash) for splash, i need to migrate expo-splash , i just waiting for better out.

**Comment by alanjhughes** [comment]
@adsalihac you will need to remove bootsplash before trying to migrate. While it is installed the expo-splash-screen plugin won't run.

**Comment by alanjhughes** [comment]
@fobos531 it's fine to leave android and ios undefined. It's not required to provide values for them. They are there if you have different needs per platform.

**Comment by fobos531** [comment]
> @fobos531 it's fine to leave android and ios undefined. It's not required to provide values for them. They are there if you have different needs per platform.

Yeah I'm aware, just highlighting that it might be useful to define the typescript type for these if they are indeed provided, in case they have different configurable options. Also, as an example from your comment here: https://github.com/expo/expo/issues/32809#issuecomment-2474876450 there is a `resizeMode` but its not listed in documentation for the plugin.

**Comment by alanjhughes** [comment]
@fobos531 I'm not following, there are types https://github.com/expo/expo/blob/9b776324d76df6cd330b00090f886030b7a8eba6/packages/expo-splash-screen/plugin/src/withSplashScreen.ts#L18-L19. If you mean autocomplete, no plugin supports it in the app.json.

**Comment by alanjhughes** [comment]
@fobos531 or do you mean in the docs? It's not currently supported by our api generation.

**Comment by fobos531** [comment]
> @fobos531 or do you mean in the docs? It's not currently supported by our api generation.

Yes, I meant in the docs. Got it, thanks for the clarification. Hopefully supported is introduced in the future.

**Comment by dedene** [comment]
> @brentvatne Seconding this! Also it seems there seems to be conflicts in icon generation on Android between splash assets and adaptive icons. If you have an adaptive icon config specified as well as a splash config via the new config plugin, those two seem to completely collide and the resulting launcher icon is the overlap of the two.

After updating from Expo 51 to 52, I was observing the same on the Android build: splash screen and icon pngs are getting mixed.

**Comment by alanjhughes** [comment]
#32908 will fix the issues related to the android icons. The changes to how you icons are defined were not intended. They should have worked as before. With this, the figma file is only outdated in relation to the splashscreen.

**Comment by vanstinator** [comment]
I love coming to a ticket to share a repro only to see that a PR is already open. Thank you!

**Comment by roni-castro** [comment]
> > @fobos531 it's fine to leave android and ios undefined. It's not required to provide values for them. They are there if you have different needs per platform.
> 
> Yeah I'm aware, just highlighting that it might be useful to define the typescript type for these if they are indeed provided, in case they have different configurable options. Also, as an example from your comment here: [#32809 (comment)](https://github.com/expo/expo/issues/32809#issuecomment-2474876450) there is a `resizeMode` but its not listed in documentation for the plugin.

@alanjhughes From what I understand, the resizeMode is currently being [hardcoded](https://github.com/expo/expo/blob/9b776324d76df6cd330b00090f886030b7a8eba6/packages/expo-splash-screen/plugin/src/withSplashScreen.ts#L30) internally for the Android default and dark mode splash screens, but not for iOS. Is there a specific reason why ios.dark [here](https://github.com/expo/expo/blob/9b776324d76df6cd330b00090f886030b7a8eba6/packages/expo-splash-screen/plugin/src/withSplashScreen.ts#L39) doesn’t have it hardcoded?

```js
const withSplashScreen: ConfigPlugin<PluginConfig> = (config, props) => {
  const android: AndroidSplashConfig = {
    ...props,
    ...props?.android,
    resizeMode: 'contain',
    dark: {
      ...props?.android?.dark,
      ...props?.dark,
      resizeMode: 'contain',
    },
  };
  const ios: IOSSplashConfig = {
    ...props,
    ...props?.ios,
    resizeMode: 'contain',
    dark: {
      ...props?.ios?.dark,
      ...props?.dark,
      resizeMode: 'contain', // shouldn't this be added to keep the same behavior of android?
    },
  };
```

**Comment by amandeepmittal** [comment]
The Figma template is now up to date and available at: https://www.figma.com/community/file/1466490409418563617/expo-app-icon-splash-v2-community.

For more info, please see: https://www.youtube.com/watch?v=3Bsw8a1BJoQ
